nebula/ui/components/MZHelpSheet.qml

```diff
@@ -96,24 +96,31 @@ MZBottomSheet {
                     Layout.topMargin: MZTheme.theme.windowMargin / 2
                     Layout.leftMargin: 8
                     Layout.alignment: Qt.AlignTop
+                    Layout.fillWidth: true
 
                     verticalAlignment: Text.AlignVCenter
                     lineHeightMode: Text.FixedHeight
                     lineHeight: 24
+                    elide: Text.ElideRight
                 }
 
                 Item {
                     Layout.fillWidth: true
                 }
 
                 MZIconButton {
-                    id: iconButton
-
                     Layout.rightMargin: MZTheme.theme.windowMargin / 2
 
                     Layout.preferredHeight: MZTheme.theme.rowHeight
                     Layout.preferredWidth: MZTheme.theme.rowHeight
 
+                    //Hacky workaround because for some reason, when opening a sheet via
+                    //the right menu button in MZMenu, this close button is in a
+                    //"Hovered" state, even though it wasn't hovered
```

Comment:
Could this be because the mouseup event that ends the hovered state is sent to the helo sheet, so the the right menu button's screen doesn't receive it, and assumes that it is still hovered?

Reply:
It is the "X" button on the help sheet that is in the hovered state when the right menu button on the source screen is clicked.

Am I misunderstanding your comment?

Reply:
Yes, I was wondering it the mouse down and up are going to two different targets, causing this issue. I don't think anything needs to be addressed in this comment.